src/Features/Traits/Eloquent.php

```diff
@@ -10,17 +10,151 @@
 trait Eloquent
 {
     /**
-     * @Given there is :count :model object in database
-     * @Given there should be :count :model object in database
-     * @Given there are :count :model objects in database
-     * @Given there should be :count :model objects in database
+     * @Given there is a model :model in the database
+     * @Given there is a :model in the database
      */
-    public function thereAreModelsInDatabase(string $model, int $count): void
+    public function thereIsAModelInTheDatabase(string $model): void
     {
         $modelClass = $this->recognizeModelClass($model);
+
+        if (!$modelClass::query()->exists()) {
+            $modelClass::factory()->create();
+        }
+
+        Assert::assertTrue($modelClass::query()->exists(), "The model $model does not exist in the database.");
```

Comment:
Oh, I don't get what's going on here.

Should this method be used to seed database or check database?